index.src.html

```diff
@@ -214,15 +256,48 @@
     <a>control messages</a> in the <a>control message queue</a>, and all pending
     callbacks.
 
-    When invoked, run the <a>Reset</a> algorithm.
+    When invoked, run the <a>Reset AudioDecoder</a> algorithm.
   </dd>
 
   <dt><dfn method for=AudioDecoder>close()</df></dt>
   <dd>
     Immediately aborts all pending work and releases system resources. Close is
     permanent.
 
-    When invoked, run the <a>Close</a> algorithm.
+    When invoked, run the <a>Close AudioDecoder</a> algorithm.
+  </dd>
+</dl>
+
+Algorithms {#audiodecoder-algorithms}
+-------------------------------------
+<dl>
+  <dt><dfn>Output AudioFrame</dfn> (with |output|)</dt>
+  <dd>
+    Run these steps:
+    1. Let |buffer| be an {{AudioBuffer}} containing the decoded audio data in
+        |output|.
+    2. Let |frame| be an {{AudioFrame}} containing |buffer| and a timestamp for
+        the output.
+    3. Invoke {{AudioDecoder/[[output callback]]}} with frame.
+  </dd>
+  <dt><dfn>Reset AudioDecoder</dfn></dt>
+  <dd>
+    Run these steps:
+    1. If {{AudioDecoder/state}} is `“closed”`, throw an {{InvalidStateError}}.
```

Comment:
Do you mean go from " -> ' ? I thought double quotes was the way WebAudio did its enum members? I tried this substitution just now including the enum definition and bikeshed gave errors

Reply:
`“` is different from `"`. The first is unicode and is a left double quotation mark, the second is an ASCII double quote. Having left double quotation mark is probably a copy-paste artifact from a word processor (that often replaces `"` in prose, because it's the correct way to quote in english), and is problematic because people won't be able to copy-paste from our spec to their code.

Reply:
Ah, good eye! Fixed.